Lib/fontTools/t1Lib/__init__.py

```diff
@@ -312,9 +314,10 @@ def findEncryptedChunks(data):
 		if eBegin < 0:
 			break
 		eBegin = eBegin + len(EEXECBEGIN) + 1
-		eEnd = data.find(EEXECEND, eBegin)
-		if eEnd < 0:
+		endMatch = EEXECEND.search(data, eBegin)
+		if not endMatch:
 			raise T1Error("can't find end of eexec part")
+		eEnd = endMatch.start()
```

Comment:
On second though, I have one minor change request:
The idiomatic way to check for None is `if endMatch is None: ...`

Reply:
Sure! Done. :smile: